.gitattributes

```diff
@@ -0,0 +1 @@
+test/data/expected/* linguist-generated
```

Comment:
If I do the type of repository in GitHub will change to "HTML", because the size of big HTML test files. I added that as a means to tell GitHub that it shall not count files in that folder in language determination. If there is another way to fix this please let me know and I can do it in an alternative way.